packages/labs/cli/src/lib/init/element-starter/templates/lib/element.ts

```diff
@@ -0,0 +1,164 @@
+import {FileTree} from '@lit-labs/gen-utils/lib/file-utils.js';
+import {
+  javascript,
+  kabobToPascalCase,
+} from '@lit-labs/gen-utils/lib/str-utils.js';
+import {Language} from '../../../../commands/init.js';
+
+export const generateElement = (
+  elementName: string,
+  language: Language
+): FileTree => {
+  const directory = language === 'js' ? 'lib' : 'src';
+  return {
+    [directory]: {
+      [`${elementName}.${language}`]:
+        language === 'js' ? js(elementName) : ts(elementName),
+    },
+  };
+};
+
+const js = (elementName: string) => {
+  const className = kabobToPascalCase(elementName);
+  return javascript`/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import { LitElement, html, css } from 'lit';
+
+/**
+ * An example element.
+ *
+ * @fires count-changed - Indicates when the count changes
+ * @slot - This element has a slot
+ * @csspart button - The button
+ */
+export class ${className} extends LitElement {
+  static get styles() {
+    return css\`
+      :host {
+        display: block;
+        border: solid 1px gray;
+        padding: 16px;
+        max-width: 800px;
+      }
+    \`;
+  }
+
+  static get properties() {
+    return {
+      /**
+       * The name to say "Hello" to.
+       * @type {string}
+       */
+      name: {type: String},
+
+      /**
+       * The number of times the button has been clicked.
+       * @type {number}
+       */
+      count: {type: Number},
+    };
+  }
+
+  constructor() {
+    super();
+    this.name = 'World';
+    this.count = 0;
+  }
+
+  render() {
+    return html\`
+      <h1>\${this._sayHello()}!</h1>
+      <button @click=\${this._onClick} part="button">
```

Comment:
Since you're showing `part`, consider also showing a custom property being used? Perhaps `h1 { font-size: --${elementName}-name-font-size;`

Reply:
Thinking about it more, I  think the name property and output are unneeded and should be removed.

Reply:
added custom prop and removed the name property but kept the DOM